src/context/Context.jsx

```diff
@@ -25,43 +26,66 @@ const ContextProvider = (props) => {
 		setResultData("");
```

Comment:
![image](https://github.com/user-attachments/assets/bd26729e-409f-4239-ab89-95e9fc1273c7)
Blocking: `delayPara` seems to be not used anywhere?